tests/Unit/Extension/CodeTransformExtra/Rpc/ExtractMethodHandlerTest.php

```diff
@@ -20,7 +25,7 @@ class ExtractMethodHandlerTest extends HandlerTestCase
     const METHOD_NAME = 'FOOBAR';
 
     /**
-     * @var ExtractMethod
+     * @var ObjectProphecy
```

Comment:
`ObjectProphecy<ExtractMethod>` if the phpstan prophecy extension is installed this will make phpstan understand it.